### PR TITLE
Run functional tests in CI

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: test/legacy/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: '2'
+
+services:
+  tests96:
+    image: postgres:9.6
+    volumes:
+    - ../../:/workspace
+    environment:
+      TBD_PGBIN: /usr/lib/postgresql/9.6/bin
```
Comment (bersace) [inline test/legacy/docker-compose.yml]: 👌

**Comment by pgiraud** [comment]
With this PR we're now able to test the agent for several versions of PG (9.4 to 10) on CentOS7.

**Comment by pgiraud** [comment]
@bersace Added the ability to install via rpm packages when running functional tests. Please review.

**Comment by bersace** [comment]
```
legacy_tests_centos7_pg10_1 exited with code 0
Aborting on container exit...
Stopping legacy_tests_centos7_pg96_1 ... 

Stopping legacy_tests_centos7_pg95_1 ... 

Stopping legacy_tests_centos7_pg94_1 ... 
```

**Pull request merged**